pyard/data_repository.py

```diff
@@ -402,7 +402,8 @@ def generate_mac_codes(db_connection: sqlite3.Connection, refresh_mac: bool):
         # Load the MAC file to a DataFrame
         mac_url = 'https://hml.nmdp.org/mac/files/numer.v3.zip'
         df_mac = pd.read_csv(mac_url, sep='\t', compression='zip',
-                             skiprows=3, names=['Code', 'Alleles'])
+                             skiprows=3, names=['Code', 'Alleles'],
+                             keep_default_na=False, na_values=['_'])
```

Comment:
Why is `na_values=['_']` necessary ? It works without it.

 ```python
>>> df_mac = pd.read_csv("numer.v3.zip", sep='\t', compression='zip', skiprows=3, names=['Code', 'Alleles'], keep_default_na=False)
>>> df_mac[df_mac['Code'] == 'NA']
      Code            Alleles
60110   NA  01/02/06/08/09/10

```

Reply:
but what if someone wants an na_value in the future?
well there there is one ready and waiting for them!